lib/administrate/namespace.rb

```diff
@@ -7,6 +7,8 @@ def initialize(namespace)
     def resources
       namespace_controller_paths.uniq.map do |controller|
         controller.gsub(/^#{namespace}\//, "").to_sym
```

Comment:
Good idea, but in that case, the block would return a nil value (we are using method `map` here) and we have to remove the nil value in the collection thereafter?